src/components/Home.css

```diff
@@ -1,14 +1,14 @@
 .Logo {
   grid-column: main;
   position: relative;
-  top: 30px;
+  top: 65px;
 }
 
 .ButtonList {
   display: flex;
   flex-direction: column;
   justify-content: space-around;
-  gap: 15px;
+  gap: 10px;
   position: relative;
-  top: 170px;
+  top: 220px;
```

Comment:
Same here, you can achieve the same results with flex without using relative positioning:
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/6980777/116576740-1f062780-a910-11eb-8405-d7902461b336.png)